amplify/backend/function/createUserNotificationsData/src/index.js

```diff
@@ -0,0 +1,89 @@
+const fetch = require('cross-fetch');
```

Comment:
Are you planning to have a single lambda handling all calls to Magicbell or multiple ones?

Reply:
There will be two separate ones, this one is needed since it uses the secret key. When we send notifications it will be done via the block ingestor so won't be in a lambda. For fetching notifications, we'll use the built in Magicbell React context, and we can optionally pass that an HMAC code which we create using the userID and our secret key, so computing that will be done in a lambda.